src/plugins/assetUploader/README.md

```diff
@@ -0,0 +1,148 @@
+# NFT Asset Uploader Guide
+
+## Pre-requisite
+
+Before you begin, make sure you have the following:
+
+1. SDK version:
+   Ensure you have version 1.9.1-tmp.0 of `@aptos-labs/ts-sdk`.
+   [SDK Package](https://www.npmjs.com/package/@aptos-labs/ts-sdk/v/1.9.1-tmp.0?activeTab=versions)
+
+2. Install the SDK with `pnpm`:
+   ```bash
+   pnpm install @aptos-labs/ts-sdk@1.9.1-tmp.0
+   ```
+
+3. Install the Irys SDK with `npm`:
+   ```bash
+   npm install @irys/sdk
+   ```
+
+4. Prepare your NFT media and metadata files:
+   Begin with the metadata template and replace the content as necessary.
+   ```json
+   {
+       "name": "Asset Name",
+       "description": "Lorem ipsum...",
+       "image": "",
+       "properties": {
+           "simple_property": "example value"
+       }
+   }
+   ```
+   Supported media types include: `png`, `jpg`, `jpeg`, `gif`, etc.
+
+   Ensure that the file name in the metadata corresponds to the media file name.
+
+   Each metadata file and corresponding NFT media should be unique.
+
+
+## How to Use
+
+Below is a sample TypeScript script that demonstrates how to use the uploader:
+
+```ts
+const example = async () => {
+    const account = Account.generate();
+    const aptosConfig = new AptosConfig({
+        network: Network.DEVNET,
```

Comment:
wait, does that work? Irys does not support Aptos devnet, need to use tesnet. SDK should throw if you configured the SDK to use testnet and trying to use Irys
take a look at https://github.com/aptos-labs/aptos-ts-sdk/blob/tmp/examples/typescript/asset_uploader.ts

Reply:
you're right, updated to testnet